extensions/runtimes/src/index.ts

```diff
@@ -353,15 +353,13 @@ export default async (composer: any): Promise<void> => {
       _project: any
     ): Promise<string> => {
       // run dotnet install on the project
-      composer.log(
-        `EXECUTE: dotnet add ${_project.name}.csproj package ${packageName}${version ? ' --version=' + version : ''} `
-      );
-      const { stderr: installError, stdout: installOutput } = await execAsync(
-        `dotnet add ${_project.name}.csproj package ${packageName}${version ? ' --version=' + version : ''}`,
-        {
-          cwd: path.join(runtimePath),
-        }
-      );
+      const command = `dotnet add ${_project.name}.csproj package "${packageName}${
```

Comment:
If there's no version number, won't this end up with an unmatched `"` after `package`?

Reply:
doh! yes.